src/python/WMCore/WMSpec/WMWorkloadTools.py

```diff
@@ -44,6 +44,20 @@ def makeList(stringList):
     raise WMSpecFactoryException("Can't convert to list %s" % stringList)
 
 
+def makeNonEmptyList(stringList):
```

Comment:
Hmm, I'm not sure we can define (cast) the spec input parameters to a WMObject (if you have an idea, I'm all ears). Perhaps what we could do is to add these functions as WMObject methods and create some sort of wrapper functions in Utils (but then we have the Utils dependency on WMCore/DataStructs).

Or I could simply move these functions to Utils/Utilities or something like that.